### PR TITLE
OCP: update rules for s390x

### PR DESCRIPTION
**Pull request by Vincent056**
Updated rules for s390x, rules got updated are:

```
file_groupowner_ovs_conf_db
file_groupowner_ovs_conf_db_lock 
file_groupowner_ovs_sys_id_conf
```
We have different uid for `/etc/openvswitch/conf.db` , `/etc/openvswitch/.conf.db.~lock~`  and `/etc/openvswitch/system-id.conf` on Z system, thus we need the update the above rules.

### CODE REVIEW

**Comment by Vincent056** [comment]
The CI failure doesn't seem like to be related to this PR, I see there is a fix coming up for the issue here: https://github.com/ComplianceAsCode/content/pull/8718

**Comment by vojtapolasek** [comment]
Hello, just FYI, you can already use combinations of already existing platforms. e.g.
platform: ocp4-node and (not s390x)
where ocp4-node and  s390x are separate platforms.
In this way, you will not duplicate code.
More info here: https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules

**Comment by Vincent056** [comment]
> Hello, just FYI, you can already use combinations of already existing platforms. e.g.
> platform: ocp4-node and (not s390x)
> where ocp4-node and s390x are separate platforms.
> In this way, you will not duplicate code.
> More info here: https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules

Aha, thanks for the info, that’s nice, I didn’t know this is already implemented, it was wip last time I looked at it. I will update the code to use it

File: linux_os/guide/services/ssh/ssh_server/sshd_disable_rhosts/tests/ocp4/e2e.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ---
-default_result: PASS
+default_result: FAIl
```
Comment (rhmdnd) [inline linux_os/guide/services/ssh/ssh_server/sshd_disable_rhosts/tests/ocp4/e2e.yml:2]: nit: FAIL

**Review by rhmdnd — CHANGES_REQUESTED** [review]
One minor change in the e2e tests.

File: shared/references/cce-redhat-avail.txt [hunk 2]
```diff
@@ -22,7 +22,6 @@ CCE-85923-1
 CCE-85924-9
 CCE-85926-4
 CCE-85927-2
-CCE-85928-0
```
[the remark this answers is not in the document]
Comment (Vincent056) [reply shared/references/cce-redhat-avail.txt:25]: just fixed that

**Comment by jhrozek** [comment]
I think the rules are fine and we can merge them, but it also looks like we have an issue with some of the sysctls checks:
```
    helpers.go:807: Result - Name: e2e-e8-master-sysctl-kernel-yama-ptrace-scope - Status: FAIL - Severity: medium
    helpers.go:814: E2E-FAILURE: The expected result for the sysctl_kernel_yama_ptrace_scope rule didn't match. Expected 'PASS', Got 'FAIL'
```
and:
```
    helpers.go:807: Result - Name: e2e-high-master-sysctl-kernel-core-pattern - Status: FAIL - Severity: medium
    helpers.go:814: E2E-FAILURE: The expected result for the sysctl_kernel_core_pattern rule didn't match. Expected 'PASS', Got 'FAIL'
```

**Comment by Vincent056** [comment]
> what do you think is a better tracker? Upstream issue here or a CMP jira?

let's merge this, I will create a Jira ticket to track sysctl, it doesn't seem the CI is passing for sysctl fix https://github.com/ComplianceAsCode/content/pull/8718

**Comment by jhrozek** [comment]
> > what do you think is a better tracker? Upstream issue here or a CMP jira?
> 
> let's merge this, I will create a Jira ticket to track sysctl, it doesn't seem the CI is passing for sysctl fix #8718

OK, thank you.

**Review by rhmdnd — APPROVED** [review]

**Comment by yuumasato** [comment]
As mentioned above, the sysctl errors in rules `sysctl_kernel_yama_ptrace_scope` and `sysctl_kernel_core_pattern` should be fixed by #8718

**Pull request merged**